third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -179,13 +181,12 @@ provider "google-beta" {
 
 The following arguments are supported:
 
-* `region` - (Required) The region the instance will sit in. Note, first-generation Cloud SQL instance
-    regions do not line up with the Google Compute Engine (GCE) regions, and Cloud SQL is not
+* `region` - (Required) The region the instance will sit in. Note, Cloud SQL is not
```

Comment:
even though this isn't a result of your change, we probably should mark `region` as Optional since that's how we treat it in other resources with this behavior